.github/workflows/pullrequest.yml

```diff
@@ -34,35 +33,34 @@ jobs:
 
     strategy:
       matrix:
-        os: [ macos-latest, windows-latest, ubuntu-latest ]
+        os: [macos-latest, windows-latest, ubuntu-latest]
 
     steps:
       - uses: actions/checkout@v2
-      - name: Cache multiple paths
-        uses: actions/cache@v2
+      - name: Setup Node.js and cache
+        uses: actions/setup-node@v2
         with:
-          path: |
-            **/node_modules
-          key: ${{ runner.os }}-${{ hashFiles('package.json') }}
-      - name: Use Node.js 12.16.2
-        uses: actions/setup-node@v1
+          node-version-file: .nvmrc
+          cache: npm
+      - name: Setup Angular cache
+        uses: actions/cache@v2
         with:
-          node-version: 12.16.2
-
+          path: node_modules/.cache
+          key: ${{ runner.os }}-pr-angular-exe-${{ hashFiles('**/package-lock.json') }}
+      - name: Setup Rust
+        run: rustup install stable
+      - name: Setup Rust cache
+        uses: Swatinem/rust-cache@v1
       - run: npm install
-      - run: npm run build:prepare
-      - run: npm run build:nexe -- -o release/out/memebox
-
-      - run: release/out/memebox --cli-test-mode=true  --config=./release/config
-
-      - run: npm run electron:build:only
-
+      - run: npm build:prepare
```

Comment:
npm run build:prepare ^^ 

probably would be easier if you try it on your repo / pr (to it)

Reply:
Just did that yesterday evening.
I'll redo the CI tasks altogether, as I want to re-add the Electron builds as well.